scripts/hardfork/build-and-test.sh

```diff
@@ -54,14 +54,24 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 if [[ $# -gt 0 ]]; then
   # Branch is specified, this is a CI run
-  ln -sf /usr/share/zoneinfo/UTC /etc/localtime
   chown -R "${USER}" /workdir
   git config --global --add safe.directory /workdir
   git fetch
-  nix-env -iA unstable.jq
-  nix-env -iA unstable.curl
-  nix-env -iA unstable.gnused
-  nix-env -iA unstable.git-lfs
+  nix-env -iA unstable.{curl,git-lfs,gnused,jq}
+
+  # Manually patch zone infos, nix doesn't provide stdenv breaking janestreet's core
```

Comment:
Is it something temporary which we will hope be fixed in the future ?

I wonder if not to put it under nix/ folder or at least extract to method which

Reply:
I think this is only a CI issue specific to nix docker images. on regular NixOS you can have proper timezone setup.

Reply:
I was thinking that the test could be run in a nix shell like this one:

https://github.com/MinaProtocol/mina/blob/ea6244d30c67283a7c4fcf28cdebf96ab5c3ede2/nix/ocaml.nix#L468-L476

which appears to have access to the `${pkgs.tzdata}` and can set the `TZDIR` properly. So that could ultimately replace the existing hack (and the modified one in this PR) if it works.